packages/idealo.design/src/server/db_Configuration/7-db.sql

```diff
@@ -0,0 +1,25 @@
+UPDATE blogposts SET previouspost where id=5 = 'contract-testing';
```

Comment:
Are you sure this line works like this? I think you might want to change it ;-)

Reply:
Thanks for the suggestion, but works fine after using the new start command for the server ;)

Reply:
Well @LaraKiyicioglu there's a typo/leftover. In the last part, it is saying `... where id=5='contract-testing'`. I'm curious if it really works and if so how. :P